lte/gateway/docker/services/c/Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN apt-get update && \
 
 COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
 RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb ${EXTRA_REPO} ${OS_RELEASE} main"
```

Comment:
Err:5 https://artifactory.magmacore.org/artifactory/debian-test focal-ci InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 9CECAD4616DD1B36
Reading package lists...
W: GPG error: https://artifactory.magmacore.org/artifactory/debian-test focal-ci InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 9CECAD4616DD1B36

Reply:
IT needs https://github.com/magma/magma/pull/12689/files to be merged